src/styles/landing-page.less

```diff
@@ -44,14 +44,19 @@
   background-color: @landing-side-bar-bg;
   color: @color-pf-white;
   @media(min-width: @screen-sm-min) {
-    flex: 0 1 auto;
-    width: 325px;
+    bottom: 0;
+    overflow: hidden;
+    overflow-y: auto;
+    position: fixed;
+    right: 0;
```

Comment:
Yes, it leaves a gutter if no scroll bar is shown. But, to me it is worse to see cutoff content and no scroll bar.